jib-gradle-plugin/README.md

```diff
@@ -99,6 +98,16 @@ For example, to build the image `my-docker-id/my-app`, the configuration would b
 jib.to.image = 'my-docker-id/my-app'
 ```
 
+### Using [Azure Container Registry (ACR)](https://azure.microsoft.com/en-us/services/container-registry/)...
```

Comment:
Header is slightly bigger than the others in this section, use `####`.

Reply:
Oops, good catch.